deployment/paiLibrary/paiService/service_management_refresh.py

```diff
@@ -67,7 +67,7 @@ def get_service_list(self):
 
     def refresh_all_label(self):
         self.logger.info("Begin to refresh all the nodes' labels")
-        machinelist = self.cluster_object_model['machinelist']
+        machinelist = self.cluster_object_model['machine']['machine-list']
```

Comment:
We are going to decouple the statically binding of `machine list`.
Could you query the machine list from k8s api?
Or via `kube get pods`?

Reply:
you should ask @ydye about this. Anyway, this PR is focus on the refresh of these services not the way of how paictl.py implement it.

Reply:
@hao1939, maybe machinelist should be kept for paiservice. it is just for paiservice, machinelist should be retrieved from k8s api, instead of clusterconfig. of course, before deploying paiservice, you will need to validate the k8s cluster state is healthy.

Reply:
We'd better remove statically machinelist, since we are working on a scalable system.
We should base on assume that system is on changing.

Reply:
My currently assume is that:
1. k8s manage the nodes, and label the nodes.
2. PAI service bind to label, for scalability, it should decouple from specific nodes when possible.

Reply:
@hao1939 , that makes sense. please present your design once ready.

Reply:
Ok, please approve this PR if appropriate. Changing to dynamic machine list can be addressed by another PR.

Reply:
ok.